driftbase/api/players/__init__.py

```diff
@@ -235,9 +235,17 @@ def endpoint_info(current_user):
         "my_gamestates": None,
         "my_player_groups": None,
         "my_summary": None,
-        "template_player_gamestate": template_player_gamestate_url,
+        "template_player_gamestate": template_player_gamestate_url
     }
 
+    url = url_for(
+        "richpresence.entry",
+        player_id=1337,
+        _external=True,
+    )
+    url = url.replace('1337', '{player_id}')
+    ret["rich_presence"] = url
```

Comment:
Self review: This probably should be called `template_richpresence` to match the other templated URLs, right?

Reply:
Yes